base/common/python/pki/nssdb.py

```diff
@@ -515,6 +515,49 @@ def add_cert(self, nickname, cert_file, token=None, trust_attributes=None):
         finally:
             shutil.rmtree(tmpdir)
 
+    def add_cert2(
```

Comment:
maybe `add_cert_new` or mark the old methos as `add_cert_old`?

Reply:
`nssdb.add_cert()` is still used in a few places, so we would have to make sure we catch all the occurrences if we chose to rename it.

Personally, I would go with renaming the existing method to `add_cert_{old,legacy,deprecated,whatever}` and let the new method inherit the name `add_cert`. Just because in the future we may not complete this refactor, and it will be confusing if there are two methods with similar names that do more or less the same thing. And yes, I do have a [comic](http://phdcomics.com/comics/archive.php?comicid=1531) for this!

I don't want to paint the bikeshed here though, call it as you see fit :laughing: